public/index.html

```diff
@@ -8,34 +8,11 @@
       name="description"
```

Comment:
The page title is now Profiili, but meta description and html lang are in English. We're going to get a complaint from the accessibility review html lang is not in sync with the meta tags and other content